ccd-definition/CaseEventToFields.json

```diff
@@ -37,11 +37,35 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseEventID": "enterGrounds",
-    "CaseFieldID": "grounds",
+    "CaseFieldID": "EPO_REASONING_SHOW",
     "PageFieldDisplayOrder": 1,
     "DisplayContext": "OPTIONAL",
     "PageID": 1,
     "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "groundsForEPO CONTAINS \"SHOW_FIELD\""
```

Comment:
I wonder what we could do to explain that this is nasty workaround. If somebody sees that piece of code 3 months later and tries to understand that code, it might end with conclusion it does not make sense and even remove it.

Maybe we should have something like `"groundsForEPO CONTAINS \"This is workaround that hides field from the UI\""` or any other explanation that makes sense.

Reply:
It can definitely be changed to something more descriptive.